conda_smithy/lint_recipe.py

```diff
@@ -122,6 +122,20 @@ def lintify(meta, recipe_dir=None):
         lints.append('The recipe `license` should not include the word '
                      '"License".')
 
+    # 11: There should be one empty line at the end of the file.
+    if recipe_dir is not None and os.path.exists(meta_fname):
+        with open(meta_fname, 'r') as f:
+            lines = f.read().split('\n')
```

Comment:
I think `splitlines` behaves a little better when it comes to Python 2/3 compatibility and unicode. However, I don't know that we will get the behavior we want then. Not sure what we should do about it.

Maybe we need start using unicode literals with the linter. That has fixed some problems in other areas. Also I expect this will fix some other linter issues with unicode that we have experienced.